src/Action.js

```diff
@@ -29,7 +29,7 @@ export const shouldHide = (props, context) =>
 
 export default createConditional({
   displayName,
-  propTypes,
+  propTypes: process.env.NODE_ENV !== 'production' ? propTypes : null,
```

Comment:
technically we could save few bytes more by changing this to smth like
```js
const spec = {
  displayName,
  shouldShow,
  shouldHide,
}

if (process.env.NODE_ENV !== 'production') {
  spec.propTypes = propTypes  
}

export default createConditional(spec)
```

but this is somewhat overdoing those already minimal size optimizations 😉